ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -41,6 +42,51 @@ module.exports = Marionette.LayoutView.extend({
   initialize: function() {
     this.$el.toggleClass('is-form-builder', this.options.isFormBuilder === true)
     this.$el.toggleClass('is-form', this.options.isForm === true)
+    this.listenTo(
+      user.getQuerySettings(),
+      'change:template',
+      this.handleTemplateChange
+    )
+    this.handleTemplateChange()
+  },
+  handleTemplateChange: function() {
+    let userDefaultTemplate = user.getQuerySettings().get('template')
+    if (userDefaultTemplate) {
```

Comment:
✏️ if you reverse the predicate, i.e. `if (!userDefaultTemplate) return;`, you can get rid of the parenthesis and nesting here.

Reply:
done